RELEASE/scripts/autoscend/iotms/mr2023.ash

```diff
@@ -868,6 +878,320 @@ void auto_handleJillOfAllTrades()
 	return;
 }
 
+boolean auto_haveEagle()
+{
+	if(auto_have_familiar($familiar[Patriotic Eagle]))
+	{
+		return true;
+	}
+	return false;
+}
+
+string activeCitZoneMod() // get the active Citizen of a Zone mods, if any
+{
+	if(!auto_haveEagle() || have_effect($effect[Citizen of a Zone]) == 0)
+	{
+		return "none";
+	}
+	visit_url("desc_effect.php?whicheffect=9391a5f7577e30ac3af6309804da6944"); // visit url to refresh Mafia's _citizenZoneMods preference
+	string activeCitZoneMod = get_property("_citizenZoneMods").to_lower_case();
```

Comment:
I think this (string check) is fine, but FYI you can get a modifier -> amount map by
```
> ash split_modifiers(get_property("_citizenZoneMods"))

Returned: aggregate string [modifier]
Hot Damage => +10
Hot Spell Damage => +10
Meat Drop => +50
Mysticality => +10
```